<?xml version="1.0"?>
<launch>
	<param name="piece_parser/bin_threshold" value="125" />

	<node pkg="jps_puzzle_piece" type="piece_parser" name="piece_parser_node" ns="piece_parser">
		<remap from="/piece_parser/input_image" to="/usb_cam/image_raw" />
	</node>

	<include file="$(find jps_puzzle_piece)/launch/hd_cam.launch" />
</launch>
